usaspending_api/accounts/migrations/0018_auto_20170322_2024.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('accounts', '0017_auto_20170320_1455'),
+    ]
+
+    tas_fk_sql = '''
+        update treasury_appropriation_account t
+        set federal_account_id = fa.id
+        from federal_account fa
+        where t.agency_id = fa.agency_identifier
+        and t.main_account_code = fa.main_account_code
+        '''
+
+    ab_fk_sql = '''
+        update appropriation_account_balances ab
+        set federal_account_id = t.federal_account_id
+        from treasury_appropriation_account t
+        where ab.treasury_account_identifier = t.treasury_account_identifier
+        '''
+
+    def insert_federal_accounts(apps, schema_editor):
+        TreasuryAppropriationAccount = apps.get_model(
```

Comment:
Interesting, I'd never seen `.get_model`.  Lets you fill the model in the same migration it's created it, I guess.  Nice!